### ISSUE TITLE
微博开屏广告域名列表

### ISSUE THREAD

**Issue by RuCu6** [blackmatrix7/ios_rule_script#590]
如题 开屏广告的域名列表如下 请去重合并到advertising.list 

HOST, h5.sinaimg.cn, reject
HOST, ipa.im.weibo.cn, reject
HOST, dp.im.wei.cn, reject
HOST, gw.im.weibo.cn, reject
HOST, gw1.im.weibo.cn, reject
HOST-SUFFIX, biz.weibo.com, reject
HOST-SUFFIX, sdkapp.uve.weibo.com, reject
HOST-SUFFIX
![E71EC2FF-5624-4E81-8D0D-8743BA244802](https://user-images.githubusercontent.com/49463634/171992076-9140d1c1-ea20-4c49-b592-c57439cd0c58.jpeg)
, ip.taobao.com, reject

**Comment by wyq977** [comment]
想问下这些域名是怎么查到的？ 我之前想屏蔽虎牙直播app的 查看log会出现很多 谢谢

**Comment by blackmatrix7** [comment]
这个屏蔽的范围看起来有点大。

因为我平时不用微博官方的客户端，还需要验证后再添加。

这个事情优先级比较低，可能要晚些时间才能处理。

**Comment by RuCu6** [comment]
是的  后面3条已经在 Advertising.list 收录了。HOST, ipa.im.weibo.cn, reject这条误杀了，其他的貌似没啥影响。

**Comment by RuCu6** [comment]
> 想问下这些域名是怎么查到的？ 我之前想屏蔽虎牙直播app的 查看log会出现很多 谢谢

Quanx查看网络活动

**Comment by blackmatrix7** [comment]
测试了一段时间没发现什么问题。

已加入，谢谢提供。
